### ISSUE TITLE
Help text in wrong column

### ISSUE THREAD

**Issue by tanthammar** [64robots/nova-fields#44]
When I use R64\NovaFields\JSON with R64\NovaFields\Text and ->help('Some text'), the help text is placed in the wrong column.
![image](https://user-images.githubusercontent.com/21239634/49033837-29082700-f1a8-11e8-9c8b-fee587e99c42.png)

```
Json::make('Settings', [
                Text::make('Bookons Unit', 'unit')
                ->help('The default unit (size) for Bookons, related to this project. Example: Meter, m2, kvm, Feet, Area… Can be overridden on individual items.')
                ,
```

**Comment by disitec** [comment]
Hi @tanthammar 

What do you mean with `placed in the wrong column`?

Original nova fields has the text beneath the label as you can see in this screenshot

![image](https://user-images.githubusercontent.com/12644599/49068543-4cbe8200-f227-11e8-95a4-ed7efa4637e7.png)

Do you propose some better location?

Thank you!

**Comment by disitec** [comment]
Please, comment if you think this is still an issue and I will reopen it. Thanks!

**Comment by tanthammar** [comment]
If I:
use Laravel\Nova\Fields\Text;
use R64\NovaFields\JSON;
I get this:
![image](https://user-images.githubusercontent.com/21239634/49371272-32891600-f6f7-11e8-9120-c13f9a3e34a0.png)

If I:
use R64\NovaFields\Text;
use R64\NovaFields\JSON;
I get this:
![image](https://user-images.githubusercontent.com/21239634/49371315-5d736a00-f6f7-11e8-8478-88583a8cd097.png)

Code:
`
 protected function settings($request)
    {
        return [
            Json::make('Settings', [
                Text::make('Bookons Unit', 'unit')
                ->help('The default unit (size) for Bookons, related to this project. Example: Meter, m2, kvm, Feet, Area… Can be overridden on individual items.')
                ,
        ]
  }
`

**Comment by disitec** [comment]
Thank you for you detailed explanation. Ill try to get this fixed ASAP

**Comment by disitec** [comment]
Fixed in 0.6.0
